app_dart/test/request_handling/authentication_test.dart

```diff
@@ -60,67 +62,6 @@ void main() {
         );
       });
 
-      test('auth succeeds with authenticated cookie but unauthenticated header', () async {
```

Comment:
These tests should have been removed in the agent cleanup. They have equivalents else where